doc/config.example.yaml

```diff
@@ -75,6 +75,43 @@ validator:
   # How often (in seconds) we should check for pre-images to reveal
   preimage_reveal_interval: 10
 
+################################################################################
+##                             Flash configuration                            ##
+## Configuration options for the Flash functionality of the node.             ##
+##                                                                            ##
+################################################################################
+flash:
+  # Whether or not the Flash protocol should be supported
+  enabled: true
+  # Timeout for each request in milliseconds
+  timeout: 10000
+  # This is a randomly generated keypair
+  # DO NOT USE THOSE VALUES ANYWHERE
+  # Private seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
+  # Public address:  GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ
```

Comment:
Outdated

Reply:
Then it means the rest of the documented config is also outdated.

Reply:
True, let's just fix it all at once